extensions/book/package.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "book",
+    "displayName": "book",
+    "description": "Support for Jupyter Books",
```

Comment:
>Books [](start = 40, length = 5)

Isn't it Jupyter Notebooks?

Reply:
Jupyter Book is a collection of notebooks and markdown files, making it easier to organize and share a bunch of notebooks.